package/gluon-config-mode-geo-location/src/de.po.in

```diff
@@ -0,0 +1,54 @@
+msgid ""
+msgstr ""
+#ifdef WITHMAP
+"Project-Id-Version: gluon-config-mode-geo-location-with-map\n"
+#else
+"Project-Id-Version: gluon-config-mode-geo-location\n"
+#endif
+"PO-Revision-Date: 2017-08-22 12:14+0100\n"
+"Last-Translator: Jan-Tarek Butt <tarek@ring0.de>\n"
+"Language-Team: German\n"
+"Language: de\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+msgid ""
+"If you want the location of your node to be displayed on the map, you can "
+"enter its coordinates here. "
+#ifdef WITHMAP
+"If your PC is connected to the internet you may also click on the map displayed below. "
+#endif
+"Please keep in mind setting a location can also enhance the network quality."
```

Comment:
that sounds pretty far fetched

Reply:
maybe: "Please keep in mind setting a location gives you a better opportunity to get contact with mesh-partners that live nearby."

Reply:
I keep this quite abstract, because it can enhance the network quality in the sense of a good public map where users can find the next node or some kind of that. Another case can be for the hoodselector which can determinate the nodes hood base on its location.